dashboard/client/src/pages/actor/index.tsx

```diff
@@ -1,4 +1,5 @@
-import { makeStyles } from "@material-ui/core";
+import { Grid, makeStyles, Switch } from "@material-ui/core";
+import dayjs from "dayjs";
```

Comment:
add to pacakge.json or do we have to use it? js date is covers majority of the use cases https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date

Reply:
Dayjs is a light-weight date library that can handle more types of time string input and more operates on time, it was add to package.json the first time new dashboard merged